commands/command_e2e_test.go

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-server/v5/api4"
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+)
+
+func (s *MmctlE2ETestSuite) TestArchiveCommandCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	teamOfBasicUser, appErr := s.th.App.CreateTeam(&model.Team{
+		DisplayName: "dn_" + model.NewId(),
+		Name:        api4.GenerateTestTeamName(),
+		Email:       s.th.BasicUser.Email,
+		Type:        model.TEAM_OPEN,
+	})
+	s.Require().Nil(appErr)
+
+	_, appErr = s.th.App.AddUserToTeam(teamOfBasicUser.Id, s.th.BasicUser.Id, "")
+	s.Require().Nil(appErr)
+
+	s.RunForSystemAdminAndLocal("Archive nonexistent command", func(c client.Client) {
```

Comment:
We can probably use `RunForAllClients` for this one too.

Reply:
fixed